events/EiffelActivityQueuedEvent/simple.json

```diff
@@ -0,0 +1,26 @@
+{
+  "meta": {
+    "domainId": "domainId-test",
+    "id": "acqueued-ec41-488e-93f2-fc2702b32f21",
```

Comment:
To be really picky this is not a valid UUID since the "q" is outside the valid range.